docs/advanced.rst

```diff
@@ -391,8 +391,13 @@ Pipenv supports the usage of environment variables in values. For example::
     maya = {version="*", index="pypi"}
     records = "*"
 
-Environment variables may be specified as ``${MY_ENVAR}`` or ``$MY_ENVAR``.
+<<<<<<< HEAD
+Environment variables is better specified as ``${MY_ENVAR}``.
```

Comment:
This isn't a stylistic choice or a suggestion, which is why we originally used the wording we did.  Here you suggest that it is "better" to specify them only one specific way, which implies that windows users are less important.  It also takes the form of a suggestion or a recommendation when in fact there is only one way to do it at all if you are a linux user, and one way if you are a windows user.  If you don't follow the specific way, your variables won't get parsed.

Hopefully that makes sense, but even if it doesn't please leave this wording as it was as @uranusjr originally asked.